dynawo/sources/API/FSV/FSVFinalStateValuesCollection.h

```diff
@@ -0,0 +1,259 @@
+//
+// Copyright (c) 2021, RTE (http://www.rte-france.com)
+// See AUTHORS.txt
+// All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at http://mozilla.org/MPL/2.0/.
+// SPDX-License-Identifier: MPL-2.0
+//
+// This file is part of Dynawo, an hybrid C++/Modelica open source time domain
+// simulation tool for power systems.
+//
+
+/**
+ * @file  FSVFinalStateValuesCollection.h
+ *
+ * @brief Final state values collection description : interface file
+ *
+ */
+
+#ifndef API_FSV_FSVFINALSTATEVALUESCOLLECTION_H_
+#define API_FSV_FSVFINALSTATEVALUESCOLLECTION_H_
+
+#include "FSVFinalStateValue.h"
+
+#include <boost/shared_ptr.hpp>
+#include <string>
+#include <vector>
+
+namespace finalStateValues {
+
+/**
+ * @class FinalStateValuesCollection
+ * @brief Final state values collection interface class
+ *
+ * Interface class for final state values collection object. This a container for final state values
```

Comment:
typo: this "is" a container for final state values

Reply:
fixed